pkg/output/logger.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
This should not be exposed to users, especially not now when this is still alpha: https://github.com/estroz/operator-sdk/commit/b6dc8ceca8e86d118d89d293e84e3bc5f5dd0d9e#r39366018.